### DISCUSSION TITLE
`Completion` gives up showing if provider takes long

### DISCUSSION THREAD

**Discussion by yitao-mutable** [microsoft/vscode-discussions#51] [Extension Development QnA] [answered]
for `Completion` or `InlineCompletion`, if the provider takes >5 seconds to resolve (after requesting an API), the text simply doesn't show up. any suggestions?

**Comment by isidorn** [comment, marked as answer]
@hediet do we have some timeout here?

**Reply by hediet** [reply]
No, I don't know where the 5 seconds come from.

**Reply by burkeholland** [reply]
We aren't sure what the answer is here, so we're going to close this as answered. If you are still experiencing the issue, please feel free to come back and re-open an unanswered.
